doc/api/next_api_changes/behavior/28354-YN.rst

```diff
@@ -0,0 +1,6 @@
+Add zorder option in QuiverKey
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+It has been needed to use ``set_zorder`` when you set zorder in the QuiverKey.
+This commit can be set zorder with ``zorder`` argument in the QuiverKey
+in the same way as other plots.
+This commit improves API consistency.
```

Comment:
```suggestion
``zorder`` can be used as a keyword argument to `.QuiverKey`. Previously,
that parameter did not have any effect because the zorder was hard coded.
```